content/docs/user-guide/gems/reference/multiplayer/multiplayer/tutorial_multiplayer.md

```diff
@@ -0,0 +1,887 @@
+---
+title: Tutorial - Your First Network Component
+linktitle: Tutorial Multiplayer
+description: Use Multiplayer Gem to create multiplayer components.
+toc: true
+---
+
+
+
+## Introduction
+
+
+This tutorial will link all the relevant documentation to provide you with a detailed guide on getting started with creating multiplayer components.
+
+Our starting base will be O3DE MultiplayerSample project. You can find the instructions on how to build and run it here: https://github.com/o3de/o3de-multiplayersample/blob/development/README.md
+
+
+
+
+
+## Network Components
+
+### Add a New Component
+
+We will start by creating a network component - MyFirstNetworkComponent. It will only do one thing, which is to replicate a monotonic counter.
+
+Since we already have MultiplayerSample setup for codegen, we can start by duplicating one of the existing codegen xml files.
+
+`D:\git\o3de-multiplayersample\Gem\Code\Source\AutoGen\SimplePlayerCameraComponent.AutoComponent.xml`
+
+into 
+
+`D:\git\o3de-multiplayersample\Gem\Code\Source\AutoGen\MyFirstNetworkComponent.AutoComponent.xml`
```

Comment:
Is this really necessary since you're giving the body of the xml below anyway? Why not just have them make the file?

Reply:
It's my personally philosophy to over provide full details in tutorials, so it's easy to go back and use it as a reference and a guide. I don't want readers to create a file and make a mistakes due to missing something in their understanding. It makes the tutorial longer but has more chances to teach new folks.